### PR TITLE
Patches

### PR DESCRIPTION
**Pull request by marwanad**
what's left is the docker auth part in test-infra in the AKS deployer and populating the env variables in the `newAKS`

### CODE REVIEW

File: cluster/images/kubemark/Makefile [hunk 1]
```diff
@@ -22,15 +22,15 @@
 REGISTRY := $(if $(REGISTRY),$(REGISTRY),staging-k8s.gcr.io)
 IMAGE_TAG := $(if $(IMAGE_TAG),$(IMAGE_TAG),latest)
 
-all: gcloudpush
+all: dockerpush
```
Comment (alexeldeib) [inline cluster/images/kubemark/Makefile]: ? why this change

Comment (marwanad) [reply cluster/images/kubemark/Makefile]: because we're going to use this as well so I find it confusing. I can revert to `gcloudpush`

Comment (alexeldeib) [reply cluster/images/kubemark/Makefile]: oh, i see how this works. makes sense
